lib/routes/v1/kyc/index.js

```diff
@@ -0,0 +1,50 @@
+'use strict'
+
+const axios = require('axios')
+const router = require('express').Router()
+const { cfHeaders = {}, kycApiUrl } = require('config')
+const { getSchemaValidationMw, checkAuth, asyncMw } = require('../../../helpers/middlewares')
+const { getBfxToken } = require('../../../helpers/grenache')
+const { auth } = require('../../../helpers/schemas')
+const { throwOnGrcErr } = require('../../../helpers/utils')
+const { GenericError } = require('../../../helpers/errors')
+
+router.post('/token/get', getSchemaValidationMw(auth), checkAuth, asyncMw(async (req, res) => {
+  const { txObj } = req.body._validated.authTx
+  const account = txObj.actions[0].authorization[0].actor
+  const result = await getBfxToken(req, account)
+
+  throwOnGrcErr(result, 'getBfxToken', 'ERR_USER')
+
+  res.status(200).json({ token: result.data.apiToken })
+}))
+
+router.post('/data/get', getSchemaValidationMw(auth), checkAuth, asyncMw(async (req, res) => {
+  const { txObj } = req.body._validated.authTx
+  const account = txObj.actions[0].authorization[0].actor
+  const result = await getBfxToken(req, account)
+
+  throwOnGrcErr(result, 'getBfxToken', 'ERR_USER')
+
+  const cplToken = result.data.cplToken
+  if (!cplToken) {
+    throw new GenericError(`grc getToken request failed: ${result.message}`)
```

Comment:
would that crash the service?

Reply:
it won't, errors are caught and processed:
https://github.com/bitfinexcom/moonbeam/blob/master/lib/helpers/middlewares.js#L11
https://github.com/bitfinexcom/moonbeam/blob/master/lib/helpers/middlewares.js#L263